main.js

```diff
@@ -0,0 +1,37 @@
+
+var suggestedFood = document.querySelector('.you-should-make');
+var letsCookButton = document.querySelector(".lets-cook-button");
+var cookpot = document.querySelector(".cookpot");
+var rightBox = document.querySelector(".right-box");
+
+
+letsCookButton.addEventListener('click', checkingRadioButtons)
+
+function checkingRadioButtons() {
+  event.preventDefault();
+  var checkedDish;
+  var foods = document.querySelectorAll("input[name='food']");
```

Comment:
Overall, it looks good. I would not have thought to do it this way, but nice work! That's the beauty of coding.
Potential Refactor - 
- think about what a <input> radio tag is, it has checked value that is true/false
- they have values, id's, names, etc.... could using querySelectors on this buttons provide more information you need to shorten your code, and potentially allow you to remove lines 12-18?

Reply:
Traditionally the name assigned to an input is for accessibility (ie. screen readers), and would want seperates names dedicated to them. 

name="sides"
name="thing"
etc.